rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -147,3 +151,104 @@ def find_exactly_one_fluid_loop(rmi, loop_id):
         loop_id,
         rmi,
     )
+
+
+def has_heating_system(rmi_b, hvac_b_id):
+    """
+    Check whether heating system exists or not.
+
+    Parameters
+    ----------
+    rmi_b json
+        To evaluate if the hvac system is modeled as Sys-2 in the B_RMI.
+
+    hvac_b_id list
```

Comment:
Isn't `hvac_b_id` a string?

Reply:
Great finding! I don't know why I thought it was a list. Fixed.